blog-homepage/src/BlogApp/CurrentArticle.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Articles from './your-articles.json';
+import Bookmark from './icons8-bookmark.svg';
+import PropTypes from 'prop-types';
+
+Articles.propTypes = {
+  src: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+}
```

Comment:
These proptypes are only assigned to components. Here, they're being assigned to the JSON object.